server/lib/handlebars.js

```diff
@@ -152,6 +152,28 @@ handlebars.registerHelper('encodeURIComponent', str => {
 
 handlebars.registerHelper('formatCurrencyObject', (obj, props) => formatCurrencyObject(obj, props.hash));
 
+handlebars.registerHelper('formatOrderAmountWithInterval', order => {
+  if (order.currency === null || order.totalAmount === null) {
```

Comment:
In case the fields are undefined

```suggestion
  if (!order.currency || !order.totalAmount) {
```